src/content/docs/en/guides/fonts.mdx

```diff
@@ -106,19 +103,24 @@ The [Fontsource](https://fontsource.org/) project simplifies using Google Fonts
     ```
 
 
+## Integrations
 
-## Add fonts with Tailwind
+### Add fonts with Tailwind
 
-If you are using the [Tailwind integration](/en/guides/integrations-guide/tailwind/), you can either add an `@font-face` statement for a local font or use Fontsource’s `import` strategy to register your font.
+If you are using the [Tailwind integration](/en/guides/integrations-guide/tailwind/), you can either add an [`@font-face` statement for a local font](#using-a-local-font-file) or use [Fontsource's `import` strategy](#using-fontsource) to register your font.
```

Comment:
```suggestion
If you are using the [Tailwind integration](/en/guides/integrations-guide/tailwind/), you can either add an [`@font-face` statement for a local font](#using-a-local-font-file) or use [Fontsource's `import` strategy](#using-fontsource) to install your font.
```
"Install" makes it clear that this is a precursor step to "register" below, not the same step